src/tests/regression/call-graph-edge-detection.test.ts

```
@@ -0,0 +1,213 @@
+  const callee = result.functions.find(f => 
+    f.name === pattern.calleeName && 
+    (!pattern.calleeFile || f.filePath.includes(pattern.calleeFile))
+  );
+  
+  const edge = caller && callee ? result.allEdges.find(e => 
+    e.callerFunctionId === caller.id && 
+    e.calleeFunctionId === callee.id
+  ) : null;
+  
+  return { edge, caller, callee };
+}
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)